llvm/test/CodeGen/AMDGPU/llvm.amdgcn.quadmask.ll

```diff
@@ -0,0 +1,90 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 2
+; NOTE: llc -march=amdgcn -mcpu=gfx1100 -amdgpu-enable-delay-alu=0 -global-isel=1 -verify-machineinstrs < %s | FileCheck  -check-prefixes=GFX11 %s
```

Comment:
Typo NOTE instead of RUN. Does the test still pass when you fix that?

Reply:
It does still work. Good catch!